modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceImplicitGrantTestCase.java

```diff
@@ -106,6 +107,7 @@ public void testInit() throws Exception {
 
 		setSystemproperties();
 		client = HttpClientBuilder.create().build();
+		scopes = "scopeTest";
```

Comment:
lets make it like `abc`
```suggestion
		scopes = "abc";
```

Reply:
Fixed